cmd/tendermint/commands/testnet.go

```diff
@@ -175,6 +178,7 @@ func testnetFiles(cmd *cobra.Command, args []string) error {
 		ChainID:         "chain-" + tmrand.Str(6),
 		ConsensusParams: types.DefaultConsensusParams(),
 		GenesisTime:     tmtime.Now(),
+		InitialHeight:   initialHeight,
```

Comment:
Do you want to add the InitialHeight argument to the cmd/init.go file for initializing individual nodes